### PR TITLE
add makefile

### PR DESCRIPTION
**Pull request by thiagormagalhaes**
Adding Makefile to the repository to run the installation, build, and use commands.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,16 @@
+TYPESCRIPT_URL = https://github.com/Microsoft/TypeScript.git \
+
+.PHONY: install
+install:
+	npm install -g typescript
+
+.PHONY: build
+build:
+	git clone $(TYPESCRIPT_URL)
```
Comment (kevinbarabash) [inline Makefile]: If this file is part of the repo they'd have to had already cloned the repo to get this file.

File: README.md [hunk 2]
```diff
@@ -93,6 +105,11 @@ jake help             # List the above commands.
 node built/local/tsc.js hello.ts
 ```
 
+or
+
+```bash
+make usage
```
Comment (kevinbarabash) [inline README.md]: `node built/local/tsc.js hello.ts` is an example of how to compile a TypeScript file in this case `hello.ts`.  There is not `hello.ts` in the repo so run `make usage` will fail.  Also, it's not very useful to compile `hello.ts` as people will probably want to compile files with different names as well.

File: Makefile (hunk 1 above)
Comment (thiagormagalhaes) [reply Makefile]: Excuse me. I had not tried that.

File: README.md (hunk 2 above)
Comment (thiagormagalhaes) [reply README.md]: Removed

**Comment by sandersn** [comment]
I don't think make support is a goal of the typescript compiler. You have to be invested in the JS ecosystem to install typescript on its own, and npm is the standard solution there.

(And more cross-platform &mdash; on this Windows laptop, I don't even *have* make, but I installed npm first thing.)

**Pull request closed without merging**